Code/GraphMol/FileParsers/FileParsers.h

```diff
@@ -34,7 +34,7 @@ class MolFileUnhandledFeatureException : public std::exception {
       : _msg(msg){};
   //! get the error message
   const char *message() const { return _msg.c_str(); };
-  ~MolFileUnhandledFeatureException() throw() override{};
```

Comment:
Was this intentional to mix into this PR?

Reply:
Its not the best place for it, but the SWIG wrappers (which I wanted to test) don't build with this form. 
Since `throw()` is deprecated anyway and this bit of code isn't yet in a release, it seemed fairly harmless. If you want, I can pull that out into its own one-line PR